src/fsharp/FSharp.Core/prim-types.fs

```diff
@@ -478,6 +478,11 @@ namespace Microsoft.FSharp.Core
         let inline isinstPrim<'T>(x:obj) = (# "isinst !0" type ('T) x : obj #)
         let inline castclassPrim<'T>(x:obj) = (# "castclass !0" type ('T) x : 'T #)
         let inline notnullPrim<'T when 'T : not struct>(x:'T) = (# "ldnull cgt.un" x : bool #)
+           
+        let inline internal notnullFast (x: ^T) = 
```

Comment:
OK, using https://github.com/Microsoft/visualfsharp/pull/1233/commits/f473a1f611ef2b934d5a7cebe0940720773793b3#diff-5470ddaa52d99663f8d391a77ae04f0dR663 seems to compile. But I'm not sure if the IL is correct ;-)